### PR TITLE
[Cache] KB Migration

### PR DESCRIPTION
**Pull request by dcpena**
Addresses [PCX-3024](https://jira.cfops.it/browse/PCX-3024).

- Migrated get started topic
- Migrated Cache Analytics topic and split it to into two, cache analytics and best practices

### CODE REVIEW

File: products/cache/src/content/about/cache-analytics.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+Cache Analytics also allows for flexible filtering of data. Create filters to focus on the traffic to optimize. Example filters include **Cache status**, **Host**, **Path**, or **Content type**.
+
+To add filters, under **Cache Performance**, click  **Add filter**.  Click **Apply** when you are done.
+
+## Review cache status
+
+The **Requests summary** graph depicts how your traffic changes over time, such as in response to a high-traffic event or a recent configuration change. Note that the Requests summary content is based on a 10% sample of requests.
+
+**Served by Cloudflare** indicates content served by Cloudflare that did not require contacting your origin web server.  **Served by Origin** indicates traffic served from the origin web server.
```
Comment (kodster28) [inline products/cache/src/content/about/cache-analytics.md]:
```suggestion
**Served by Cloudflare** indicates content served by Cloudflare that did not require contacting your origin web server. **Served by Origin** indicates traffic served from the origin web server. 
```

File: products/cache/src/content/about/cache-analytics.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+To add filters, under **Cache Performance**, click  **Add filter**.  Click **Apply** when you are done.
+
+## Review cache status
+
+The **Requests summary** graph depicts how your traffic changes over time, such as in response to a high-traffic event or a recent configuration change. Note that the Requests summary content is based on a 10% sample of requests.
+
+**Served by Cloudflare** indicates content served by Cloudflare that did not require contacting your origin web server.  **Served by Origin** indicates traffic served from the origin web server. 
+
+For **Data Transfer**, **Revalidated** requests are considered **Served by Cloudflare**. However, revalidated requests count as **Served by Origin** within the **Requests **view. This analytics behavior reflects that Cloudflare must check the origin web server for revalidated cache requests before returning a result from cache.
```
Comment (kodster28) [inline products/cache/src/content/about/cache-analytics.md]:
```suggestion
For **Data Transfer**, **Revalidated** requests are considered **Served by Cloudflare**. However, revalidated requests count as **Served by Origin** within the **Requests** view. This analytics behavior reflects that Cloudflare must check the origin web server for revalidated cache requests before returning a result from cache.
```

File: products/cache/src/content/about/cache-analytics.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+**Cache status** graphs help explain why traffic is served from Cloudflare versus the origin web server. The graph shows analytics by content-type to portray how different components of your website perform:
+
+For a list of cache statuses and their descriptions, refer to [Cloudflare cache responses](/about/default-cache-behavior#cloudflare-cache-responses).
+
+## Review requests by source
+
+Cache Analytics shows top metrics (Top-N) for several request components. Apply filters before reviewing Top-N metrics. For example, filtering to only view traffic with an Expired or Revalidated Cache status allows review of which URLs were primarily responsible for those statuses.
```
Comment (kodster28) [inline products/cache/src/content/about/cache-analytics.md]:
```suggestion
Cache Analytics shows top metrics (Top-N) for several request components. Apply filters before reviewing Top-N metrics. For example, filtering to only view traffic with an Expired or Revalidated Cache status lets you review which URLs were primarily responsible for those statuses.
```

File: products/cache/src/content/best-practices/cache-performance.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+pcx-content-type: how-to
+---
+
+# Cache performance
+
+## Optimize cache ratios
+
+Depending on the cache status you receive, you can make modifications to improve your cache ratio. To review the list of cache statuses, refer to [Cloudflare cache responses](/about/default-cache-behavior#cloudflare-cache-responses).
+
+- **Dynamic**: Default response for many file types including HTML. To cache additional content, refer to [custom caching with page rules](https://support.cloudflare.com/hc/en-us/articles/360021023712-Best-Practices-Speed-up-your-Site-with-Custom-Caching-via-Cloudflare-Page-Rules).
```
Comment (kodster28) [inline products/cache/src/content/best-practices/cache-performance.md]:
```suggestion
- **Dynamic**: Default response for many file types including HTML. To cache additional content, refer to [custom caching with page rules](https://support.cloudflare.com/hc/articles/360021023712).
```

File: products/cache/src/content/best-practices/cache-performance.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+   - Identify the issue: Click **Add filter** and select **Cache status equals Dynamic**. 
+   - Resolution: Set a Cloudflare Page Rule to [cache dynamic content](/best-practices/customize-cache).
+
+- Short cache expiration TTL.
+   - Identify the issue: Click **Add filter** and select **Cache status equals Revalidated**. 
+   - Resolution: [Increase Cloudflare’s Edge Cache TTL via a Page Rule](/about/edge-browser-cache-ttl).
+
+- Need to enable Tiered Cache or Custom Cache Key 
+  - Identify the issue: Click **Add filter** and select **Cache status equals Miss**. 
+  - Resolution: Enable Argo Tiered Caching or [create a custom cache key](https://developers.cloudflare.com/cache/about/cache-keys).
```
Comment (kodster28) [inline products/cache/src/content/best-practices/cache-performance.md]:
```suggestion
  - Resolution: Enable Argo Tiered Caching or [create a custom cache key](/about/cache-keys).
```

File: products/cache/src/content/get-started.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+- [What is a CDN?](https://developers.cloudflare.com/fundamentals/get-started/cdn)
+- [Understand the default file types Cloudflare caches](https://developers.cloudflare.com/cache/about/default-cache-behavior#default-cached-file-extensions)
+
+## Make more resources cacheable
+
+Configure your settings to cache static HTML or cache anonymous page views of dynamic content.
+
+- [Speed up your site with caching best practices](https://support.cloudflare.com/hc/en-us/articles/360021023712)
+- [Specify which resources to cache](https://developers.cloudflare.com/cache/best-practices/customize-cache)
```
Comment (kodster28) [inline products/cache/src/content/get-started.md]:
```suggestion
- [Specify which resources to cache](/best-practices/customize-cache)
```

File: products/cache/src/content/get-started.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+- [Understand Origin Cache Control](https://developers.cloudflare.com/cache/about/cache-control)
```
Comment (kodster28) [inline products/cache/src/content/get-started.md]:
```suggestion
- [Understand Origin Cache Control](/about/cache-control)
```

File: products/cache/src/content/get-started.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+- [Cache by device type (Enterprise only)](https://developers.cloudflare.com/cache/how-to/create-page-rules#cache-by-device-type-enterprise-only)
```
Comment (kodster28) [inline products/cache/src/content/get-started.md]:
```suggestion
- [Cache by device type (Enterprise only)](/how-to/create-page-rules#cache-by-device-type-enterprise-only)
```

File: products/cache/src/content/get-started.md [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+
+## Improve cache hit rates
+
+Include or exclude query strings, optimize cache keys, or enable tiered cache to improve hit rates and reduce traffic to your origin.
+
+- [Choose a cache level](https://developers.cloudflare.com/cache/how-to/set-caching-levels)
```
Comment (kodster28) [inline products/cache/src/content/get-started.md]:
```suggestion
- [Choose a cache level](/how-to/set-caching-levels)
```

File: products/cache/src/content/get-started.md [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+- [Enable Tiered Cache with Argo](https://support.cloudflare.com/hc/en-us/articles/115000224552)
```
Comment (kodster28) [inline products/cache/src/content/get-started.md]:
```suggestion
- [Enable Tiered Cache with Argo](https://support.cloudflare.com/hc/articles/115000224552)
```

File: products/cache/src/content/get-started.md [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+- [Configure custom cache keys (Enterprise only)](https://developers.cloudflare.com/cache/about/cache-keys)
```
Comment (kodster28) [inline products/cache/src/content/get-started.md]:
```suggestion
- [Configure custom cache keys (Enterprise only)](/about/cache-keys)
```

File: products/cache/src/content/get-started.md [hunk 12]
```diff
@@ -0,0 +1,69 @@
...
+
+## Secure your cache configuration
+
+Control resources a client is allowed to load and set access permissions to allow different origins to access your origin’s resources. Protect your site from web cache deception attacks while still caching static assets.
+
+- [Avoid web cache poisoning attacks](https://developers.cloudflare.com/cache/best-practices/avoid-web-poisoning)
```
Comment (kodster28) [inline products/cache/src/content/get-started.md]:
```suggestion
- [Avoid web cache poisoning attacks](/best-practices/avoid-web-poisoning)
```

File: products/cache/src/content/get-started.md [hunk 13]
```diff
@@ -0,0 +1,69 @@
...
+- [Configure Cross-Origin Resource Sharing (CORS)](https://support.cloudflare.com/hc/en-us/articles/200308847)
```
Comment (kodster28) [inline products/cache/src/content/get-started.md]:
```suggestion
- [Configure Cross-Origin Resource Sharing (CORS)](https://support.cloudflare.com/hc/articles/200308847)
```

File: products/cache/src/content/get-started.md [hunk 14]
```diff
@@ -0,0 +1,69 @@
...
+- [Enable Cache Deception Armor](https://blog.cloudflare.com/web-cache-deception-attack-revisited/)
```
Comment (kodster28) [inline products/cache/src/content/get-started.md:39]: Should there be another ticket to move this blog content into dev docs?

File: products/cache/src/content/get-started.md [hunk 15]
```diff
@@ -0,0 +1,69 @@
...
+To provide Cloudflare services to our customers, we may need to alter your HTML or cached objects to enable the feature or provide optimization.
+
+These code alterations only occur on the cacheable objects found at Cloudflare's edge and do not affect the original source. The changes will also be removed if the specific feature is disabled and the cache is purged.
+
+Review the list of Cloudflare features that function in this manner.
+
+- [Auto Minify](https://support.cloudflare.com/hc/en-us/articles/200168196-How-do-I-minify-HTML-CSS-and-JavaScript-to-optimize-my-site-)
+- [Rocket Loader](https://support.cloudflare.com/hc/en-us/articles/200168056-What-does-Rocket-Loader-do-)
+- [Polish](https://developers.cloudflare.com/images/polish)
+- [Mirage](https://support.cloudflare.com/hc/en-us/articles/219178057-Configuring-Cloudflare-Mirage)
```
Comment (kodster28) [inline products/cache/src/content/get-started.md]:
```suggestion
- [Mirage](https://support.cloudflare.com/hc/articles/219178057)
```

File: products/cache/src/content/get-started.md [hunk 16]
```diff
@@ -0,0 +1,69 @@
...
+
+## Cloudflare features that can alter your HTML and cacheable objects
+
+To provide Cloudflare services to our customers, we may need to alter your HTML or cached objects to enable the feature or provide optimization.
+
+These code alterations only occur on the cacheable objects found at Cloudflare's edge and do not affect the original source. The changes will also be removed if the specific feature is disabled and the cache is purged.
+
+Review the list of Cloudflare features that function in this manner.
+
+- [Auto Minify](https://support.cloudflare.com/hc/en-us/articles/200168196-How-do-I-minify-HTML-CSS-and-JavaScript-to-optimize-my-site-)
```
Comment (kodster28) [inline products/cache/src/content/get-started.md]:
```suggestion
- [Auto Minify](https://support.cloudflare.com/hc/articles/200168196)
```

File: products/cache/src/content/get-started.md [hunk 17]
```diff
@@ -0,0 +1,69 @@
...
+- [Rocket Loader](https://support.cloudflare.com/hc/en-us/articles/200168056-What-does-Rocket-Loader-do-)
```
Comment (kodster28) [inline products/cache/src/content/get-started.md]:
```suggestion
- [Rocket Loader](https://support.cloudflare.com/hc/articles/200168056)
```

File: products/cache/src/content/get-started.md (hunk 15 above)
Comment (kodster28) [inline products/cache/src/content/get-started.md]:
```suggestion
- [Mirage](https://support.cloudflare.com/hc/articles/219178057)
```

File: products/cache/src/content/get-started.md [hunk 18]
```diff
@@ -0,0 +1,69 @@
...
+- [Polish](https://developers.cloudflare.com/images/polish)
+- [Mirage](https://support.cloudflare.com/hc/en-us/articles/219178057-Configuring-Cloudflare-Mirage)
+- [Hotlink Protection](https://support.cloudflare.com/hc/en-us/articles/200170026-Understanding-Cloudflare-Hotlink-Protection)
```
Comment (kodster28) [inline products/cache/src/content/get-started.md]:
```suggestion
- [Hotlink Protection](https://support.cloudflare.com/hc/articles/200170026)
```

File: products/cache/src/content/get-started.md [hunk 19]
```diff
@@ -0,0 +1,69 @@
...
+- [Email address obfuscation](https://support.cloudflare.com/hc/en-us/articles/200170016-What-is-Email-Address-Obfuscation-)
```
Comment (kodster28) [inline products/cache/src/content/get-started.md]:
```suggestion
- [Email address obfuscation](https://support.cloudflare.com/hc/articles/200170016)
```

File: products/cache/src/content/get-started.md [hunk 20]
```diff
@@ -0,0 +1,69 @@
...
+- [Bot Management Javascript Detections](https://developers.cloudflare.com/bots/reference/javascript-detections)
+
+<Aside type="note" header="Note:">
```
Comment (kodster28) [inline products/cache/src/content/get-started.md]: Shouldn't this say submit an issue to our github repo? Or do we even still need this?

File: products/cache/src/content/get-started.md [hunk 21]
```diff
@@ -0,0 +1,69 @@
...
+
+The list above may not be complete. If you think a product or feature should be added, add a post in the [Community](https://community.cloudflare.com/).
+
+</Aside>
+
+## Troubleshoot
+
+Resolve common caching concerns.
+
+- [Learn about Cloudflare's cache response statuses](https://developers.cloudflare.com/cache/about/default-cache-behavior#cloudflare-cache-responses)
```
Comment (kodster28) [inline products/cache/src/content/get-started.md]:
```suggestion
- [Learn about Cloudflare's cache response statuses](/about/default-cache-behavior#cloudflare-cache-responses)
```

File: products/cache/src/content/get-started.md [hunk 22]
```diff
@@ -0,0 +1,69 @@
...
+- [Investigate Cloudflare's cache response with cURL](https://support.cloudflare.com/hc/en-us/articles/203118044#h_0c7f48b3-fc29-4266-8c63-477fe61a11c4)
```
Comment (kodster28) [inline products/cache/src/content/get-started.md]:
```suggestion
- [Investigate Cloudflare's cache response with cURL](https://support.cloudflare.com/hc/articles/203118044#h_0c7f48b3-fc29-4266-8c63-477fe61a11c4)
```

File: products/cache/src/content/get-started.md [hunk 23]
```diff
@@ -0,0 +1,69 @@
...
+- [Diagnose Always Online issues](https://developers.cloudflare.com/cache/best-practices/always-online)
```
Comment (kodster28) [inline products/cache/src/content/get-started.md]:
```suggestion
- [Diagnose Always Online issues](/best-practices/always-online)
```

**Review by kodster28 — APPROVED** [review]

File: products/cache/src/content/get-started.md (hunk 14 above)
Comment (dcpena) [reply products/cache/src/content/get-started.md:39]: I had this same thought! I will most definitely create a ticket for that.

File: products/cache/src/content/get-started.md (hunk 20 above)
Comment (dcpena) [reply products/cache/src/content/get-started.md]: Good callout. I'm leaning towards removing it and will do that

**Pull request merged**